clockify_client/models/task.py

```diff
@@ -16,44 +16,43 @@ def add_task(
         workspace_id: str,
         project_id: str,
         task_name: str,
-        request_data: dict | None = None,
+        payload: dict | None = None,
     ) -> JsonType:
         """
         Creates new task in Clockify.
 
         :param workspace_id  Id of workspace.
-        :param request_data  Dictionary with request data.
+        :param payload       Dictionary with request data.
         :param project_id    Id of project.
         :param task_name     Name of new task.
         :return              Dictionary with task object representation.
         """
-        url = f"{self.base_url}/workspaces/{workspace_id}/projects/{project_id}/tasks/"
-        payload = {"name": task_name, "projectId": project_id}
-        if request_data:
-            payload = {**payload, **request_data}
+        path = f"/workspaces/{workspace_id}/projects/{project_id}/tasks/"
 
-        return self.post(url, payload)
+        _payload = {"name": task_name, "projectId": project_id}
+        payload = {**_payload, **payload} if payload else _payload
```

Comment:
Don't dictionaries have an extend method like lists do? You could just `_payload.extend(payload)` here and then pass that in to the post below.

Reply:
I can use `update` instead of splatting.